projects/rulenode-core-config/src/lib/components/action/gps-geo-action-config.component.ts

```diff
@@ -28,9 +28,14 @@ export class GpsGeoActionConfigComponent extends RuleNodeConfigurationComponent
   rangeUnits = Object.keys(RangeUnit);
   rangeUnitTranslationMap = rangeUnitTranslations;
 
+  presenceMonitoringStrategies = PresenceMonitoringStrategiesData;
+  presenceMonitoringStrategyKeys = Array.from(this.presenceMonitoringStrategies.keys());
+
   timeUnits = Object.keys(TimeUnit);
   timeUnitsTranslationMap = timeUnitTranslations;
 
+  public defaultPaddingEnable = true;
```

Comment:
What is the variable of this variable if it does not change?

Reply:
I have added the logic for this variable